### ISSUE TITLE
TypeError: Meteor.Collection.getAll is not a function

### ISSUE THREAD

**Issue by sebastianconcept** [abecks/meteor-infinite-scroll#6]
I'm reaching this exception in a `METEOR@1.3.2.4` app.

Throws it at:
https://github.com/abecks/meteor-infinite-scroll/blob/master/infinite-scroll.js#L75

I suggest:

```
  const collectionExists = !!global[options.collection] && (global[options.collection] instanceof Meteor.Collection);
    // Collection exists?
  if (!collectionExists) {
    throw new Error('Collection does not exist: ', options.collection);
  } else {
    //set collection to cursor. collectionExists.name evaluates to a string
    collection = global[options.collection];
  }
```

**Comment by JulianKingman** [comment]
I'm getting a similar error, but from uglify, like this: https://github.com/meteor/meteor/issues/5387

```
.../parse.js:196:18:   Unexpected token name (collectionExists)
```

**Comment by abecks** [comment]
Will try to get this fixed. Short on time right now. Will accept PRs.

**Comment by JulianKingman** [comment]
Made PR

**Comment by abecks** [comment]
#7 merged and 0.9.1 pushed to Atmosphere. Please let me know if there are any issues!

**Comment by fligen** [comment]
Im still getting this issue.

TypeError: Meteor.Collection.getAll is not a function
    at Blaze.TemplateInstance.infiniteScroll (infinite-scroll.js:75)
    at Blaze.TemplateInstance.<anonymous> (categories.js:21)
    at template.js:119
    at Function.Template._withTemplateInstanceFunc (template.js:465)
    at fireCallbacks (template.js:115)
    at Blaze.View.<anonymous> (template.js:208)
    at view.js:107
    at Object.Blaze._withCurrentView (view.js:538)
    at view.js:106
    at Object.Tracker._runFlush (tracker.js:511)

**Comment by TheCodeLord** [comment]
Yep, I got the same issue

**Comment by abecks** [comment]
I'll take a look at this today or tomorrow, feel free to submit a PR in the meantime

**Comment by TheCodeLord** [comment]
Thanks, to specify my issue:

In Mozilla:
Exception in template helper: Blaze.TemplateInstance.prototype.parents@http://localhost:3000/packages/jwo1f_parent-template.js?hash=343a9da27d5234d375ef3579638ae1e6774bf36e:53:15

In Chrome:
Exception in template helper: TypeError: Cannot read property '_templateInstance' of null
    at Blaze.TemplateInstance.parents (http://localhost:3000/packages/jwo1f_parent-template.js?hash=343a9da27d5234d375ef3579638ae1e6774bf36e:53:28)

**Comment by mizsha** [comment]
https://github.com/dburles/mongo-collection-instances/issues/32

**Comment by mizsha** [comment]
https://github.com/abecks/meteor-infinite-scroll/pull/12

**Comment by TheCodeLord** [comment]
I found my problem, I made a subscription for the collection and it conflicted with the one that this library is setting. After the subscription was deleted everything works like it is supposed to.

P.S. : The name of the collection (in this.infiniteScroll{ collection: 'name' }) should be as the name of your collection in Mongo not as the name of your collection in the model, in the model my collection started with capital case and in Mongo with lower, this throws: "Error: Collection does not exist: "
